config/schedule.rb

```diff
@@ -15,4 +15,7 @@
 
   # make sure the daemon is running
   rake "ts:start"
-end
\ No newline at end of file
+
+  # Clean up expired media
+  runner "MediaCleanupJob.perform_later"
```

Comment:
I'm not sure about the rake task, the index option on the rake interface takes a few options from the task. I wouldn't want to duplicate & recreate that portion of TS's task code. Also not big on interacting with sphinx through an internal API, could be an update headache if they decide to refactor that code.

The choice to run that hourly was more or less entirely arbitrary, mostly because there was other stuff already in that block in the schedule. It could run less frequently, or more frequently (but with something other than cron). I know there's at least one option for scheduling ActiveJobs that uses the after callback to do the re-enqueuing, and that option wouldn't spin up an additional rails process.